packages/module/patternfly-docs/content/extensions/react-log-viewer/demos/ComplexToolbarLogViewer.jsx

```diff
@@ -1,239 +0,0 @@
-import React from 'react';
```

Comment:
Just curious, why did you elect to move this from a separate file back into the markdown file?

Reply:
I moved all the files back now to seperate files.